Makefile

```diff
@@ -223,7 +223,7 @@ release/versionbump:
 
 .PHONY: release/image
 release/image:
-	bazel clean --expunge
+	# bazel clean --expunge
```

Comment:
We added this because otherwise we ended up with a stale base image.  Have we resolved that issue another way?

Reply:
We have the correct stamp in the Workspace file, so the stamp will update it now

Reply:
@rail can you please confirm this

Reply:
I'm not quite sure how to confirm this... What did exactly change?

Reply:
Yeah we do not a checksum/digest in the WORKSPACE file, so I will add this back in, but it kills dev time ;(. Will file an issue.

Reply:
I think the issue is [this thing](https://github.com/cockroachdb/cockroach-operator/blob/a65d8e9dcb100035010bbd8ec294529bb29837a7/cmd/cockroach-operator/BUILD.bazel#L37-L45), which is not deterministic. I'm not sure how to make it deterministic. :/